strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/MavenArtifactGenerator.java

```diff
@@ -429,4 +430,10 @@ public void setLicenses(MavenLicenseConfiguration[] licenses)
         this.licenses = licenses;
     }
 
+    @Override
+    public void setLicenses(LicenseConfiguration[] licenses)
+    {
+
+    }
```

Comment:
I thought I already implemented this for the `MavenArtifactGenerator`...?

Reply:
Yes, we need to override it as `MavenArtifactGenerator.java` and specific implementation of `LicenseConfiguration` i.e. `MavenLicenseConfiguration`

Reply:
I'm sorry, I don' think I understand why this is required. Could you please try to explain in more details? I seem to be missing something. Each `ArtifactGenerator` implementation should have its own annotation. At least that was how I had implemented it.

Reply:
In this case, I suggest adding this, so that we don't break it for Maven:
```suggestion
    public void setLicenses(LicenseConfiguration[] licenses)
    {
        this.licenses = licenses;
    }
```

Reply:
Changed as suggested.